bitnami/mariadb-galera/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: mariadb-galera
-version: 3.1.3
+version: 3.1.5
```

Comment:
Since you are adding a new feature this should be a minor version

Reply:
Well, it is backwards-compatible, but fine. Done.